velox/docs/functions/spark/datetime.rst

```diff
@@ -299,6 +299,14 @@ These functions support TIMESTAMP and DATE input types.
 
         SELECT timestamp_millis(1230219000123); -- '2008-12-25 15:30:00.123'
 
+.. spark:function:: timestamp_seconds(x) -> timestamp
+
+    Returns timestamp from the number of seconds since UTC epoch.
+    Supported types are: TINYINT, SMALLINT, INTEGER, BIGINT, FLOAT, and DOUBLE.::
```

Comment:
Could you please update the doc to describe how float and double are handled and also provide examples?

Reply:
Thank you, I updated the doc. There turned out to not be any difference in the treatment of floats and doubles after adopting the approach of first scaling to microseconds. This mirrors and matches Spark as well as being simpler and more consistent (i.e. `timestamp_seconds(double(1.1234567))` == `timestamp_seconds(float(1.1234567))`).